lib/chai/core/assertions.js

```diff
@@ -814,23 +814,32 @@ module.exports = function (chai, _) {
    *     expect(undefined).to.be.undefined; // Recommended
    *     expect(undefined).to.not.exist; // Not recommended
    *
+   * Use after `.property` to assert that a value is not `null` nor `undefined`.
+   *     expect({a: 1}}).to.have.property('a').that.exists;
```

Comment:
I'm not sure I like this example. For the most part, we discourage the use of `.exists` in favor of a precise assertion whenever reasonable. For example:

```js
expect({a: 1}).to.have.property('a', 1); // Recommended
expect({a: 1}).to.have.property('a').that.equals(1); // Recommended
expect({a: 1}).to.have.property('a').that.exists; // Not recommended
```